core/services/privacy/erasure/PersonalDataEraserInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace EventEspresso\core\services\privacy\erasure;
+
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
```

Comment:
this needs to be removed because we are more strictly following PSR now which does not allow any side effects in files that defines symbols (classes, interfaces, etc)

You can run code sniffer to see all of the issues that will cause branches to fail on Travis now

Reply:
ya oups